### PR TITLE
Fixed overflow style on Tabpanel for Safari

### PR DESCRIPTION
**Pull request by arorayash**

### CODE REVIEW

File: src/common/styles/GlobalStyles.tsx [hunk 1]
```diff
@@ -50,6 +50,16 @@ const GlobalStyles = createGlobalStyle`
     overscroll-behavior-x: none; /* Prevent the page redirection on swipe */
   }
 
+  ::-webkit-scrollbar {
```
Comment (Rohanhacker) [inline src/common/styles/GlobalStyles.tsx]: don't add it to global we just need it for tab

File: src/common/styles/GlobalStyles.tsx [hunk 2]
```diff
@@ -50,6 +50,16 @@ const GlobalStyles = createGlobalStyle`
...
+    width: 0px;
```
Comment (Rohanhacker) [inline src/common/styles/GlobalStyles.tsx]: also set 
  scrollbar-width: none; /* Firefox 64 */
  -ms-overflow-style: none; /* Internet Explorer 11 */

Comment (arorayash) [reply src/common/styles/GlobalStyles.tsx]: Done.

File: src/common/styles/GlobalStyles.tsx (hunk 1 above)
Comment (arorayash) [reply src/common/styles/GlobalStyles.tsx]: Yeah, had added it for testing on the diff device first, updated now.

**Comment by arorayash** [comment]
test this please

**Pull request merged**